ygen/gogen.go

```diff
@@ -546,6 +546,10 @@ func (E_{{ .EnumerationPrefix }}) IsYANGGoEnum() {}
 // ΛMap returns the value lookup map associated with  {{ .EnumerationPrefix }}.
 func (E_{{ .EnumerationPrefix }}) ΛMap() map[string]map[int64]ygot.EnumDefinition { return ΛEnum; }
 
+func (e E_{{ .EnumerationPrefix }}) String() string {
```

Comment:
Can we add a comment to this function to ensure that it's understood that this function will return an error string embedded in the output - and therefore the output might not be the enumeration's actual string value? We really need users *not* to call this when doing rendering of this value to a string.

Reply:
Added:
// String returns a logging-friendly string for E_{{ .EnumerationPrefix }}.